node/src/reactor/main_reactor/tests.rs

```diff
@@ -1869,3 +1811,38 @@ async fn run_redelegate_bid_network() {
     // redelegated bid exists
     check_bid_existence_at_tip(&net, &charlie.1, Some(&alice.1), true);
 }
+
+// 1 -> 90
+// 2 -> 150
+#[tokio::test]
+async fn rewards_are_calculated() {
```

Comment:
Maybe it's worth adding some more complex testing around rewards too. Do you think it's worth adding a scenario where some delegation happens?

Reply:
Alex L is taking care of that.